### PR TITLE
DTGB-717: Added language switcher.

### PR DESCRIPTION
**Pull request by delrueba**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/00-settings/_vars.scss [hunk 1]
```diff
@@ -64,6 +64,7 @@ $z-layers: (
   'default': 1,
   'skiplink': 5,
   'mijn-gent': 99,
+  'language-switcher': 99,
```
Comment (hstandaert) [inline components/00-settings/_vars.scss]:
```suggestion
  'language-switcher': 6,
```

File: components/00-settings/_vars.scss [hunk 2]
```diff
@@ -64,6 +64,7 @@ $z-layers: (
   'default': 1,
   'skiplink': 5,
   'mijn-gent': 99,
```
Comment (hstandaert) [inline components/00-settings/_vars.scss]:
```suggestion
  'mijn-gent': 6,
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: components/00-settings/_vars.scss (hunk 2 above)
Comment (meirege) [reply components/00-settings/_vars.scss]: Are you sure about this @hstandaert ?

Comment (delrueba) [reply components/00-settings/_vars.scss]: I might have to update them again when making the mobile menu anyway.

**Review by hstandaert — APPROVED** [review]

**Pull request merged**